### PR TITLE
WP - Display "End Date" and "Closed By" on the initiative close out item

### PR DESCRIPTION
**Pull request by ajaitasaini**
### Description

This PR displays information about the close out on the initiative details dashboard. Also fixes a broken id in the json that linked the projected end date in close out to the projected end date set in define initiative.

### Related ticket(s)

CMDCT-226

---
### How to test

- The easier way to test this is by pulling this PR locally, and commenting out `disabled={entityStatus === "disabled"}` in `EntityRow.tsx`. Create an initiative in the work plan, fill out steps 1-4, and close out the initiative. When you return to the dashboard you should see close out details. 
- The _correct_ way to test this would be to fill out an entire WP as a state user, log in as an admin, approve / unlock the WP, re log in as the state user, close out the initiative, and see the close out details.
- Should match design here: https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=516-69318&mode=design&t=eGxVvwlg03mWq7Dg-4

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/tables/EntityRow.tsx [hunk 1]
```diff
@@ -29,12 +29,13 @@ export const EntityRow = ({
   openAddEditEntityModal,
   openDeleteEntityModal,
   openOverlayOrDrawer,
+  stepType,
```
Comment (ailZhou) [inline services/ui-src/src/components/tables/EntityRow.tsx:32]: just a slight recommendation to cut down on adding new props, you can "formEntity.stepType" to get step type information

**Review by ailZhou — DISMISSED** [review]
Besides that one rec, everything works well and close up text shows up in the entity. Not a big deal if you don't swap it.

**Review by cassandradanger — COMMENTED** [review]
talked about this, but posting here to keep track - couldn't see the close out information when I closed out the initiative

**Review by britt-mo — APPROVED** [review]
Saw the close-out initiatives section enabled after commenting out 
`disabled={entityStatus === "disabled"}`

**Review by ailZhou — APPROVED** [review]

**Pull request merged**